types/tar/index.d.ts

```diff
@@ -198,14 +202,18 @@ export const knownExtended: {
 export const headerSize: number;
 export const blockSize: number;
 
+interface ParseConstructor {
+  new (opt?: any): ParseStream;
```

Comment:
This shouldn't be `any`

Reply:
This doesn't seem to match up with the constructor options from the tar docs: https://github.com/npm/node-tar#constructoroptions-2